noir-projects/aztec-nr/aztec/src/keys/point_to_symmetric_key.nr

```diff
@@ -0,0 +1,32 @@
+use dep::protocol_types::{constants::GENERATOR_INDEX__SYMMETRIC_KEY, grumpkin_point::GrumpkinPoint, utils::arr_copy_slice};
+use dep::std::{hash::sha256, grumpkin_scalar::GrumpkinScalar, scalar_mul::variable_base_embedded_curve};
+
+// TODO(#5726): This function is called deriveAESSecret in TS. I don't like point_to_symmetric_key name much since
```

Comment:
The `deriveAESSecret` as a name will be a pain with the outgoing as it derives an AES secret, but not from the point. `secretAndPointToSymmetricKey` incoming 😆